drivers/audio/cxd56.c

```diff
@@ -1312,15 +1312,15 @@ static void _process_audio_with_src(cxd56_dmahandle_t hdl, uint16_t err_code)
     {
       FAR struct ap_buffer_s *src_apb;
 
-      src_apb = (struct ap_buffer_s *) dq_get(&dev->down_runq);
+      src_apb = (struct ap_buffer_s *)dq_get(&dev->down_runq);
```

Comment:
```suggestion
      src_apb = (FAR struct ap_buffer_s *)dq_get(&dev->down_runq);
```

Reply:
Let's keep as before since:

1. All pointer doesn't add FAR
2. It's specific to Spresense(arm core)